packages/test-cases/cases/wrappers/wasm-as/simple-subinvoke/invoke/src/index.ts

```diff
@@ -1,5 +1,10 @@
 import { Args_add, Add_Module } from "./wrap";
+import { Args_add as ImportedArgs_add } from "./wrap/imported/Add_Module/serialization";
 
-export function add(args: Args_add): string {
-  Add_Module.add
+export function add(args: Args_add): i32 {
+  let importedArgs: ImportedArgs_add = {
+    a: args.a,
+    b: args.b
+  }
+  return Add_Module.add(importedArgs).unwrap()
 }
```

Comment:
Why this change? I'd prefer to have a different output for invoke and subinvoke. It'll be easy to debug and spot some horrible bugs that would be passed safely due to a buggy client implementation.

Reply:
I changed it just to make it work and have a super simple example

>  I'd prefer to have a different output for invoke and subinvoke

Gotcha, if we just use the `add` method in the `invoke` and add `1` to that result, would that be enough for you? Meaning that the `invoke` method would return the sum of two numbers + 1, and the `subinvoke` would just return the sum of two numbers

Reply:
Yeah, that'd work. You can call it `add_and_increament` then